ScintSuite/LibData/AUG/Misc.py

```diff
@@ -229,7 +229,7 @@ def get_icrh(shot: int, coupled: bool = False):
     """
     ICP = sf.SFREAD(shot, 'ICP')
     if coupled:
-        name = 'PICRNc'
+        name = 'PICRFc'
```

Comment:
Please leave it as it was, if for your shot there was not signal because they cahnged the name, just make a try, try with the current one and if fails, use the new one, but if this new name does not exist for an old shot, you will make other people code to fail and worst, if it does exist and it means other thing you can create a bug extremeally complex to find